src/SearchResults.js

```diff
@@ -0,0 +1,41 @@
+import moment from 'moment';
+import SearchRow from './SearchRow';
+
+function SearchResults(props) {
+  
+  let tr = props.results.map((element) => (
+    <SearchRow
+       key = {element.id}
```

Comment:
Nice, you have remembered to use a key! 😄

Reply:
I do remember